UserApi/UserApi/Helper/UserProfileHelper.cs

```diff
@@ -25,6 +25,12 @@ public class UserProfileHelper
             {"Civil Money Claims", AdGroup.MoneyClaims}
         };
 
+        private static readonly Dictionary<AdGroup, string> CaseTypeMappings = new Dictionary<AdGroup, string>
```

Comment:
IDictionary ?

Reply:
Since it's both instantiated and used only internally in the class, is there a benefit in generalising the member type?

Reply:
No benefit, but just good to work with the abstraction no ??

Reply:
Well, yagni to me but I don't mind either way. Updated it.